### PR TITLE
[testenv] Address novaNames replacements on todo

### PR DESCRIPTION
**Pull request by bogdando**
Use novaNames and cellXNames for other places from the todo list.

### CODE REVIEW

File: test/functional/base_test.go [hunk 1]
```diff
@@ -484,23 +489,24 @@ func GetCellNames(novaName types.NamespacedName, cell string) CellNames {
...
-	APIConfigDataName               types.NamespacedName
-	InternalAPINetworkNADName       types.NamespacedName // refers internalapi network, not Nova API
+	Namespace               string
+	NovaName                types.NamespacedName
+	InternalNovaServiceName types.NamespacedName
+	PublicNovaServiceName   types.NamespacedName
+	AdminNovaServiceName    types.NamespacedName
+	InternalNovaRouteName   types.NamespacedName
+	PublicNovaRouteName     types.NamespacedName
+	AdminNovaRouteName      types.NamespacedName
```
Comment (mrKisaoLamb) [inline test/functional/base_test.go:499]: this also can be removed

**Review by mrKisaoLamb — CHANGES_REQUESTED** [review]
small comment and need rebase. Except that lgtm

**Review by gibizer — COMMENTED** [review]
good direction. I think we can easily land this after the conflict is resolved.

**Review by mrKisaoLamb — COMMENTED** [review]
I rebase and IMO can be merged

**Review by gibizer — APPROVED** [review]
Looks good.

**Pull request merged**